rpm/native_db.go

```diff
@@ -59,9 +59,22 @@ func packagesFromDB(ctx context.Context, pkgdb string, db nativeDB) ([]*claircor
 			PackageDB: pkgdb,
 		})
 		p := &ps[idx]
-		if strings.Contains(info.Module, ":") {
-			p.Module = info.Module
+		var modStream string
+		if strings.Count(info.Module, ":") > 1 {
+			first := true
+			idx := strings.IndexFunc(info.Module, func(r rune) bool {
+				if r != ':' {
+					return false
+				}
+				if first {
+					first = false
+					return false
+				}
+				return true
+			})
+			modStream = info.Module[:idx]
```

Comment:
Is there ever a case where `if strings.Count(info.Module, ":") == 1` we just want to keep the string?

Reply:
That should never happen, and the previous `rpm`-output handling code ignored the field in that case.